src/components/dialogs/BcolErrorDialog.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-dialog :value="bcolDialog" width="45rem" persistent :attach="attach" content-class="bcol-error-dialog">
+    <v-card v-if="bcolObject">
+      <v-card-title id="dialog-title">Payment Incomplete - {{bcolObject.title}}</v-card-title>
+
+      <v-card-text>
+        <p class="genErr" id="dialog-header">This {{filingTypeToName(filingType)}} could not be filed for the following reason:</p>
```

Comment:
Line too long.

In the future, have a look at your own files (https://github.com/bcgov/business-filings-ui/pull/75/files) and watch for these. These should be triggering lint errors but I don't know why they're not.

Reply:
done...and yeah I didn't have any linting errors

Reply:
Maybe those warnings show up if you do `npm run build` ?

Reply:
Fixed.